data/mods/Magiclysm/materials.json

```
@@ -237,5 +237,22 @@
     "name": "Mana",
     "copy-from": "pseudo_fuel",
     "fuel_data": { "energy": 500, "perpetual": true }
+  },
+  {
+    "type": "material",
+    "id": "concentrated_mana",
+    "name": "Concentrated Mana",
+    "density": 38,
+    "reinforces": true,
```

```suggestion
    "reinforces": false,
```
and isn't there also a "no repair" thing? there's no way you can repair this.